test/suites/clustering.sh

```diff
@@ -3678,3 +3678,60 @@ test_clustering_events() {
   kill_lxd "${LXD_FOUR_DIR}"
   kill_lxd "${LXD_FIVE_DIR}"
 }
+
+test_clustering_uuid() {
+  # shellcheck disable=2039,3043
+  local LXD_DIR
+
+  setup_clustering_bridge
+  prefix="lxd$$"
+  bridge="${prefix}"
+
+  # create two cluster nodes
+  setup_clustering_netns 1
+  LXD_ONE_DIR=$(mktemp -d -p "${TEST_DIR}" XXX)
+  chmod +x "${LXD_ONE_DIR}"
+  ns1="${prefix}1"
+  spawn_lxd_and_bootstrap_cluster "${ns1}" "${bridge}" "${LXD_ONE_DIR}"
+
+  cert=$(sed ':a;N;$!ba;s/\n/\n\n/g' "${LXD_ONE_DIR}/cluster.crt")
+
+  setup_clustering_netns 2
+  LXD_TWO_DIR=$(mktemp -d -p "${TEST_DIR}" XXX)
+  chmod +x "${LXD_TWO_DIR}"
+  ns2="${prefix}2"
+  spawn_lxd_and_join_cluster "${ns2}" "${bridge}" "${cert}" 2 1 "${LXD_TWO_DIR}"
+
+  ensure_import_testimage
+
+  # spawn an instance on the first LXD node
+  LXD_DIR="${LXD_ONE_DIR}" lxc launch testimage c1 --target=node1
+  # get its volatile.uuid
+  uuid_before_move=$(LXD_DIR="${LXD_ONE_DIR}" lxc query /1.0/instances/c1 | jq -r '.config."volatile.uuid"')
+  # stop the instance
+  LXD_DIR="${LXD_ONE_DIR}" lxc stop c1
```

Comment:
`lxc stop -f c1` please so we dont get any chance of a hanging test here.

Reply:
Also does this container ever get deleted?

Reply:
line 3725, it is deleted

Reply:
oh in node1 ? No, you're right